nfsn-hook.sh

```diff
@@ -25,4 +26,4 @@ function unchanged_cert {
     echo " + Certificate for ${DOMAIN} unchanged."
 }
 
-HANDLER=$1; shift; $HANDLER $@
+"$@"
```

Comment:
Again, for clarity I prefer the longer version, though `$@` should be quoted for sanity.